packages/strapi-plugin-i18n/admin/src/hooks/useAddLocale/index.js

```diff
@@ -22,12 +23,21 @@ const addLocale = async ({ code, name, isDefault }) => {
 
     return data;
   } catch (e) {
-    strapi.notification.toggle({
-      type: 'warning',
-      message: { id: 'notification.error' },
-    });
-
-    return e;
+    const message = get(e, 'response.payload.message', null);
+
+    if (message && message.includes('already exists')) {
```

Comment:
Why don't we simply remove the already added locales from the list instead ?

Reply:
I don't feel comfortable making this kind of computations, I'm feeling this is more a backend thing on here.

This is a fix on the frontend that will hopefully (let's cross the fingers 🤞🏻 ) move to the backend